src/cmdlib.sh

```diff
@@ -515,3 +515,18 @@ run_virtinstall() {
                                            --ostree-repo="${ostree_repo}" "$@"
     mv "${tmpdest}" "${dest}"
 }
+
+get_latest_build() {
+    if [ -L builds/latest ]; then
+        readlink builds/latest
+    fi
+}
+
+get_latest_qcow() {
```

Comment:
Utterly minor nit; I'd call this `get_latest_qemu()` as the qcow is part is less important than targeting qemu.

Reply:
Done! :arrow_down: